EDDNResponder/EDDNResponder.cs

```diff
@@ -258,6 +260,7 @@ private bool LocationIsSet()
                     ((IDictionary<string, object>)faction).Remove("SquadronFaction");
```

Comment:
I would probably introduce a variable so I only have to do the cast once, for readability. Not a big deal though.